nodejs/ot-shim/client/client.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const {
+  lightstep,
+  opentelemetry,
+} = require('lightstep-opentelemetry-launcher-node');
+const opentracing = require('opentracing');
+const { TracerShim } = require('@opentelemetry/shim-opentracing');
+
+const TARGET_URL = process.env.TARGET_URL || 'http://localhost:8080/ping';
+const sdk = lightstep.configureOpenTelemetry();
+
+// development purposes
```

Comment:
did you intend to leave this in?

Reply:
yes in development this was easier - you can't debug the docker